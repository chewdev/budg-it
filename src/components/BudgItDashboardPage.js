import React from 'react';
import ExpenseList from './ExpenseList';
import ExpenseListFilters from './ExpenseListFilters';
import BudgItSummary from './BudgItSummary';
import IncomeList from './IncomeList';

const BudgItDashboardPage = () => (
	<div>
		<BudgItSummary />
		<ExpenseListFilters />
		<ExpenseList />
		<IncomeList />

	</div>
);

export default BudgItDashboardPage;