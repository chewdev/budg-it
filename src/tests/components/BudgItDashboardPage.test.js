import React from 'react';
import { shallow } from 'enzyme';
import BudgItDashboardPage from '../../components/BudgItDashboardPage';

test('should properly render the BudgItDashboardPage component', () => {
    const wrapper = shallow(<BudgItDashboardPage />);
    expect(wrapper).toMatchSnapshot();
});