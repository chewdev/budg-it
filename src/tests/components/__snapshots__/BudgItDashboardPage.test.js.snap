// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should properly render the BudgItDashboardPage component 1`] = `
<div>
  <Connect(ExpenseListFilters) />
  <Connect(ExpenseList) />
  <Connect(IncomeList) />
</div>
`;
